### PR TITLE
Clean up DB env config

### PR DESCRIPTION
**Pull request by kylape**
- Hard coded db engine (for simplicity's sake)
- Standardized DB env var names

Needs to be merged along with https://github.com/RedHatInsights/e2e-deploy/pull/1109

### CODE REVIEW

**Comment by kylape** [comment]
uhhh I pushed a commit that isn't showing up on this PR

File: .env.example [hunk 1]
```diff
@@ -1,12 +1,12 @@
 DATABASE_SERVICE_NAME=POSTGRES_SQL
```
Comment (coderbydesign) [inline .env.example]: We could probably pull this out as well?

File: tox.ini [hunk 2]
```diff
@@ -21,8 +21,8 @@ setenv =
   DATABASE_SERVICE_NAME={env:DATABASE_SERVICE_NAME:POSTGRES_SQL}
```
Comment (coderbydesign) [inline tox.ini]: We could probably pull this out as well?

**Comment by coderbydesign** [comment]
@kylape I added @chambridge to this and the e2e PR for some visibility if he's able to give us a few to call out any red flags since he has some context. 😄

**Comment by chambridge** [comment]
Changes are fine. Legacy skeleton brought over from other projects.

**Pull request merged**